src/components/ui/ListEntry.js

```diff
@@ -10,6 +10,8 @@ const Icons = styled.div`
   & > *:not(:last-child) {
     margin: 0 8px;
   }
+  margin-right: ${(props) => (props.prepend ? '18px' : 'none')};
+  margin-left: ${(props) => (!props.prepend ? '18px' : 'none')};
```

Comment:
Maybe write this as:

`${(props) => (props.prepend ? 'margin-right' : 'margin-left')}: 18px;`

Reply:
Also it's best to change `prepend` to `$prepend` here and on 54 and 59! They're called transient props in styled components.

Reply:
Do you mean like "props.$prepend" ?